### PR TITLE
Removed the redundant code

### PR DESCRIPTION
**Pull request by annu12340**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -16,17 +16,15 @@ const App = () => {
...
+      const idx = items.indexOf(key)
+      // Remove the element.
+      items.splice(idx, 1);
+      
       // Update the state.
       setSelectedItems(items);
     } else {
       // Add the element at the end and sort the whole array alphabetically and update the state.
-      setSelectedItems([...SelectedItems, key].sort((a, b) => +a - +b));
+      setSelectedItems([...SelectedItems, key];
```
Comment (praveenscience) [inline src/components/App.js]: We need a `.sort()`, but the callback function is not necessary.

File: src/components/App.js [hunk 2]
```diff
@@ -16,17 +16,15 @@ const App = () => {
       // Make a copy of the selected items to not mutate the state.
       let items = [...SelectedItems];
       // Find index at which the element exists.
-      const idx = items.indexOf(key);
-      // If the element is there, then...
-      if (idx > -1) {
-        // Remove the element.
-        items.splice(idx, 1);
-      }
+      const idx = items.indexOf(key)
```
Comment (praveenscience) [inline src/components/App.js]: `;` missing.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Small changes. 😊

File: src/components/App.js (hunk 2 above)
Comment (annu12340) [reply src/components/App.js]: Oh!
My bad :grimacing:

**Review by praveenscience — APPROVED** [review]
Looks good to me! 👍🏻

**Pull request merged**